apps/pano/app/routes/api/auth/discord.callback.tsx

```diff
@@ -0,0 +1,10 @@
+// app/routes/auth/discord.callback.tsx
+import type { LoaderFunction } from "@remix-run/node";
+import { authenticator } from "~/authenticator.server";
+
+export let loader: LoaderFunction = ({ request }) => {
+  return authenticator.authenticate("discord", request, {
```

Comment:
i think we are missing `await` here

Reply:
Neither the example for auth strategies nor @usirin 's code includes await for this component, should I add it anyways?